stroeer/pages/section/v1/section_page.proto

```diff
@@ -13,12 +13,67 @@ message SectionPage {
 }
 
 message Stage {
-  repeated stroeer.core.v1.Article articles = 1;
+  repeated StageItem stage_items = 1;
   StageConfiguration stage_configuration = 2;
-  repeated Stage stages = 3;
 }
 
+/* possible problems:
+  - linkout as 'string' isn't in sync with the actual section tree
+  - where to derive position from for the 'linkout'?
+*/
 message StageConfiguration {
+  /* stage labels
+     e.g.: ["title":"Coronavirus", "subtitle":"Alle News und Informationen"]
+     and overrides
+     e.g.: ["title_size":"SMALL", "style/theme/palette":"COLOR_BLARZ", "logo_position:center", "qcu":"true", "swipeable":"true"]
+  */
   map<string, string> fields = 1;
-  string layout = 2;
+  StageLayout stage_layout = 2;
 }
+
+// wip: common lingo discussion planned
+enum StageLayout {
+  STAGE_LAYOUT_UNSPECIFIED = 0;
+  STAGE_LAYOUT_STORIES = 1;
+  STAGE_LAYOUT_BULLETS = 2;
+}
+
+message StageItem {
+  oneof item_type {
+    Stage stage = 1;
+    ArticleTeaser article_teaser = 2;
+    Commercial commercial = 3;
+  }
+  reserved 4 to 10; // for future item types
+  StageItemWidth stage_item_width = 11;
```

Comment:
move this to index `1` and let the `oneof` occupy the other indices

Reply:
reserving seems to be meant for deleted fields: https://developers.google.com/protocol-buffers/docs/proto3#reserved